### ISSUE TITLE
Issue(s) with usage example: fixture 'mqtt_settings' not found

### ISSUE THREAD

**Issue by aaronkurz** [mqtt-tools/pytest-mqtt#20]
I am encountering issues when trying to execute the basic example. 

I am running pytest with this test:
```python
from pytest_mqtt.model import MqttMessage
import pytest


@pytest.mark.capmqtt_decode_utf8
def test_mqtt_send_receive(mosquitto, capmqtt):
    """
    Basic send/receive roundtrip, using text payload (`str`).

    By using the `capmqtt_decode_utf8` marker, the message payloads
    will be recorded as `str`, after decoding them from `utf-8`.
    Otherwise, message payloads would be recorded as `bytes`.
    """
    # Submit a basic MQTT message.
    capmqtt.publish(topic="foo", payload="bar")

    # Demonstrate the "messages" property.
    # It returns a list of "MqttMessage" objects.
    assert capmqtt.messages == [
        MqttMessage(topic="foo", payload="bar", userdata=None),
    ]

    # Demonstrate the "records" property.
    # It returns tuples of "(topic, payload, userdata)".
    assert capmqtt.records == [
        ("foo", "bar", None),
    ]
```

However, I get the error 
```
file /home/aaron/Developer/factory-activity-detection/mqtt_xes/venv/lib/python3.10/site-packages/pytest_mqtt/mosquitto.py, line 83
  @pytest.fixture(scope="session")
  def mosquitto(mqtt_settings: MqttSettings):
E       fixture 'mqtt_settings' not found
```

How can I fix this issue?

Furthermore, I noticed that `import pytest` is missing for the basic usage example.

**Comment by amotl** [comment]
Hi. We just yanked the release 0.4.0 from PyPI. Apologies for the delay.

**Comment by amotl** [comment]
pytest-mqtt 0.4.1 has been released and verified on behalf of https://github.com/daq-tools/lorrystream/pull/95. Closing this. Thanks again.

**Comment by amotl** [comment]
> Furthermore, I noticed that `import pytest` is missing for the basic usage example.

Thanks. Fixed with 3e410fb2cff.
